content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+--> Templates` and open the template that should be modified. Click :guilabel:`Reset to draft` to
+edit the template. The dynamic variable(s) need to be configured on the Odoo side as well to specify
+the field that should be referenced.
+
+To edit the dynamic variables on a template, first change the :guilabel:`Type` to :guilabel:`Field
+of Model`. This allows Odoo to reference a field within a model to produce unique data in the
+end message.
+
+Next, go to the :guilabel:`Variables` tab and edit the :guilabel:`Field` field of the dynamic
+variables. The :guilabel:`Applies to` field can be edited prior to ensure the correct model and
+field are referenced.
+
+.. tip::
+   By default Meta uses the :guilabel:`Applies to` field set to the :guilabel:`Contact` model.
+
+To search the available fields, simply type in the front-end name (on user-interface) of the field
+in the search. This will find a result from all of the available fields for the model
+(:guilabel:`Applies to`) that the template is created for. There may be multiple levels that need to
+be configured.
+
+.. warning::
+   Customizing WhatsApp templates is out of the scope of Odoo Support.
+
+After updating the dynamic variables in the template it needs to be submitted to Meta for approval
+once again. Click :guilabel:`Submit for Approval` to start the approval process. An email will be
+sent to the administrator of the Meta account when the template has been approved.
+
+Following the approval from Meta, the templates need to be synced once again on the Odoo database.
+See this documentation: :ref:`whatsapp/sync`.
+
+.. tip::
+   Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` made
+   produce an error. This should always result in `Phone` or `Mobile`.
+
+.. seealso::
+   For more information on Meta templates visit `Meta's WhatsApp comprehensive template
+   documentation
+   <https://developers.facebook.com/docs/whatsapp/business-management-api/message-templates/>`_.
+
+Notifications
+=============
+
+Notifications in WhatsApp are handled similar to a message conversation in Odoo. A pop-up window
+will appear with the received conversation from the customer. By default notifications are set in
+the WhatsApp business account configuration in Odoo.
+
+Navigate to :menuselection:`WhatsApp app --> Configuration --> WhatsApp Business Accounts`. Select
+the account and scroll down to the :menuselection:`Control` section where notifications are handled.
+Under :guilabel:`Notify users` type in the user(s) that should be notified for this particular
+WhatsApp channel.
+
+.. tip::
+   Set up multiple WhatsApp accounts for different departments. For example, the helpdesk team and
+   sales team can chat on different channels.
+
+.. note::
+   Once a conversation is initiated between a user and a customer, notifications to all the users
+   specified in the WhatsApp business account configuration won't take place. Only notifications to
```

Comment:
```suggestion
   specified in the WhatsApp business account configuration cannot be modified for the current conversation. Only notifications to
```

I was confused by this and unsure what you meant- I believe once the conversation starts, the notifications can't be modified/ Is that true? If t, just clarify what can and can't be modified and when.

Reply:
the notifications won't occur to other users